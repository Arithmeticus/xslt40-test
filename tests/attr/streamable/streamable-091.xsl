<?xml version="1.0" encoding="UTF-8"?>
<xsl:transform xmlns:xsl="http://www.w3.org/1999/XSL/Transform" 
    xmlns:xs="http://www.w3.org/2001/XMLSchema"
    exclude-result-prefixes=" xs"
    version="3.0">
       
  
  <!-- within a streaming template, xsl:value-of concatenating text nodes 
       from within and outwith the streamed document -->
  
   
  <xsl:mode name="s" streamable="yes"/>
  <xsl:strip-space elements="*"/>
       
  <xsl:output method="xml" indent="yes" encoding="UTF-8" />
    
  <xsl:template name="main" match="/">
    <out>
      <xsl:apply-templates select="doc('loans.xml')" mode="s"/>
    </out>
  </xsl:template>
  
  <xsl:variable name="t" as="text()">
    <xsl:text>##</xsl:text>
  </xsl:variable>
  
  <xsl:template match="Property" mode="s">
      <IsCalifornia><xsl:value-of select="$t, Address/State, $t"/></IsCalifornia>
  </xsl:template>
  
  <xsl:template match="text()" mode="#all"/>
       
</xsl:transform>

