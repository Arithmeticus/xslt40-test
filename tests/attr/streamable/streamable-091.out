<?xml version="1.0" encoding="UTF-8"?>
<out>
   <IsCalifornia>## VA ##</IsCalifornia>
   <IsCalifornia>## TX ##</IsCalifornia>
   <IsCalifornia>## MD ##</IsCalifornia>
   <IsCalifornia>## CA ##</IsCalifornia>
   <IsCalifornia>## AZ ##</IsCalifornia>
</out>